### ISSUE TITLE
[Dúvida] sobre o certificado

### ISSUE THREAD

**Issue by alissonsilvas** [DenysXavier/WSBoletoSantander#53]
Olá @DenysXavier 

Esse certificado utilizado .PEM é o mesmo A1 ? 
A biblioteca ainda continua sendo utilizada em produção?

**Comment by DenysXavier** [comment]
> A biblioteca ainda continua sendo utilizada em produção?

Sim. Embora, ela não esteja sendo atualizada no ritmo pretendido.

> Esse certificado utilizado .PEM é o mesmo A1 ?

.pem é o arquivo, não tem relação direta com a terminologia A1.

Para fazer comunicação com os WebServices do Santander, seu certificado digital tem que apresentar duas característica: 

1. Comprovar sua identidade para um computador remoto
2. Sua chave pública deve ser de, pelo menos, 2048 bits.

Isso independente se for A1 ou A3.

**Comment by alissonsilvas** [comment]
Olá @DenysXavier muito obrigado pelas respostas, já testei e dei uma olhada no código dá lib ela é muito boa parabéns! no meu projeto precisei fazer um fork para adequar a minha necessidade que era adicionar o campo sacador/avalista caso tenha interesse posso mandar um pull request, possa ser que mais alguém no futuro tenha a mesma necessidade. :-)

**Comment by DenysXavier** [comment]
Boa tarde, @AlissonSilvaCe!

Você vai acabar me achando muito puritano, mas pretendo manter o cliente WS o mais aderente possível com a especificação do Santander.

Se eu pudesse te recomendar algo, recomendaria que tentasse levar esse assunto para o próprio Santander para que incluam esse campo em versões futuras.

Abraços!
